### ISSUE TITLE
Projects fail to load in VS without having built on command line first

### ISSUE THREAD

**Issue by nguerrera** [dotnet/corefx#1091]
Since the new resources targets were added, projects cannot be opened in VS from a clean enlistment with no packages having been restored yet.

**Repro steps**

1. Delete corefx/packages/ folder
2. Open a solution in VS

```
C:\src\corefx\src\System.Reflection.Metadata\src\System.Reflection.Metadata.csproj : error  :
 The imported project "C:\src\corefx\packages\Microsoft.DotNet.BuildTools.1.0.25-prerelease-00015
 \lib\resources.targets" was not found. Confirm that the path in the <Import> declaration 
 is correct, and that the file exists on disk.  C:\src\corefx\src\dir.targets
```

**Comment by nguerrera** [comment]
Assigning to @tarekgh /cc @weshaggard

**Comment by weshaggard** [comment]
Yes we need to add the exists condition around the import for resources.

**Comment by tarekgh** [comment]
assigned to @weshaggard  as he is already have the fix.

**Comment by ellismg** [comment]
@weshaggard Was this included in #1117?

**Comment by weshaggard** [comment]
Yes this is fixed.
